test/baseline/queries/pick/PickSurface.txt

```diff
@@ -126,7 +126,8 @@ d:  <zonal>
 
 K:  curv2d.silo
 curvmesh2d 
-Point (Z,R): <-2.19513, 3.43955>
+Point (Z,R): (in transformed space)
```

Comment:
Why did these tests change? Are we no longer able to do the inverse transform of the coordinates?

Reply:
The points are actually the same as before, they now just have a message about being in transformed space. It's not entirely clear why, but I think it might have something to do with adding transformations that weren't defined before. I'll do some further digging.

Reply:
So, these results are coming from the "Surface" plot, and I have no idea what this is. I can't seem to access or find it anywhere in VisIt.

Reply:
Fixed with last commit.